<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Constructor</class>
 <widget class="QWidget" name="Constructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QTabWidget" name="tabBio">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Конструктор</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="gen_ratio_button">
               <property name="text">
                <string>Сгенерировать рацион</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>262</width>
              <height>313</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>180</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Тут будет название рецепта</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="font">
                  <font>
                   <italic>true</italic>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Ингридиенты:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Много-много текста
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_20">
                 <property name="font">
                  <font>
                   <italic>true</italic>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>Способ приготовления</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="how_cook">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Много-много текста
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Рецепты</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>255</width>
            <height>315</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QListView" name="recipes_list">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>165</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="dragDropOverwriteMode">
              <bool>false</bool>
             </property>
             <property name="layoutMode">
              <enum>QListView::SinglePass</enum>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="selectionRectVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="offer_recipe_button">
             <property name="text">
              <string>Предложить рецепт</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>195</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>193</width>
            <height>315</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QLineEdit" name="lineEdit">
             <property name="inputMask">
              <string>Введите часть названия</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="text">
                  <string>Ккал</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Белков</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="text">
                  <string>Жиров</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>Углеводов</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <widget class="QLabel" name="label_28">
                 <property name="text">
                  <string>от</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_29">
                 <property name="text">
                  <string>от</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_30">
                 <property name="text">
                  <string>от</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_31">
                 <property name="text">
                  <string>от</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QSpinBox" name="min_cal"/>
               </item>
               <item>
                <widget class="QSpinBox" name="min_p"/>
               </item>
               <item>
                <widget class="QSpinBox" name="min_f"/>
               </item>
               <item>
                <widget class="QSpinBox" name="min_p_2"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QLabel" name="label_32">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_33">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_34">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_36">
                 <property name="text">
                  <string>до</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <widget class="QSpinBox" name="max_cal"/>
               </item>
               <item>
                <widget class="QSpinBox" name="max_p"/>
               </item>
               <item>
                <widget class="QSpinBox" name="max_f"/>
               </item>
               <item>
                <widget class="QSpinBox" name="max_c"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Частота</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="sort_box">
               <item>
                <property name="text">
                 <string>От популярного к редкому</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>От редкого к популярному</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="filter_button">
             <property name="text">
              <string>Применить фильтр</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Профиль</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Логин:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Цель:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Вес (кг):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Активность:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Рост (см):</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Пол:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Возраст:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Права:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Калорий в день:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Белков  в день:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Жиров  в день:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Углеводов  в день:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="user_login_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_goal_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_lweight_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_activity_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_height_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_sex_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_years_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_right_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_cal_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_p_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_f_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="user_c_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="Update_info">
             <property name="text">
              <string>Обновить информацию</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Log_out">
             <property name="text">
              <string>Выйти</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
